### PR TITLE
fastem-gui: chamber tab improvements

### PR DESCRIPTION
**Pull request by philipwinklerdelmic**
* better icon for ebeam button
* chamber button: don't show pressure on the button, it's confusing

### CODE REVIEW

File: src/odemis/gui/cont/microscope.py [hunk 1]
```diff
@@ -138,6 +139,9 @@ def __init__(self, btn_ctrl, va, main_data):
         if model.hasVA(main_data.chamber, "pressure"):
             main_data.chamber.pressure.subscribe(self._on_pressure_change, init=True)
 
+        # TODO: support alternative pressure label (e.g. TextControl below button)?
```
Comment (Sabrina85) [inline src/odemis/gui/cont/microscope.py:142]: How do you show the pressure now, if not on the button?

Comment (philipwinklerdelmic) [reply src/odemis/gui/cont/microscope.py:142]: I don't show it, it's either pumped (green) or vented (black). Do you think it's a good idea to also show the numerical value?

Comment (Sabrina85) [reply src/odemis/gui/cont/microscope.py:142]: Ok. From the TODO I thought there is still somewhere the pressure shown, as it says "alternative pressure label". I don't think we show the pressure in the SPARC, but not sure. Easy check you can do with the simulator. Job would be the right person to say if we should display it or not. Maybe you can rephrase your TODO or delete it if Job says it is not needed.

Comment (philipwinklerdelmic) [reply src/odemis/gui/cont/microscope.py]: I updated the PR to show the pressure in a label below the buttons.

File: src/odemis/gui/cont/microscope.py [hunk 2]
```diff
@@ -94,13 +94,14 @@ def Enable(self, enabled=True):
...
-    def __init__(self, btn_ctrl, va, main_data, show_pressure=True):
+    def __init__(self, btn_ctrl, va, main_data, pressure_ctrl=None):
         """
 
         :param btn_ctrl: (ImageTextToggleButton) Button that controls and displays the chamber state
         :param va: (VigillantAttribute) The chamber state
         :param main_data: (MainGUIData) GUI microscope model
-        :param show_pressure (bool): whether to show the numerical pressure value on the button
+        :param pressure_ctrl (wxStaticText, None): control for showing the numerical pressure value.
+            If None, show the pressure on the button.
```
Comment (Sabrina85) [inline src/odemis/gui/cont/microscope.py:104]: Shouldn't it be shown at all, when it is None? For the SecomStateController you had before that the pressure is simply not shown at all if I understood correctly. And now you will show the pressure on top of the button, which is not the standard behavior (just pump/vent labels).

We have now defined two behaviors:
1. The pressure is shown as text below the pump/vent button.
2. No pressure is shown, just the pump/vent button as usual.

Comment (philipwinklerdelmic) [reply src/odemis/gui/cont/microscope.py:104]: The behaviour before (for secom I guess?) was to show the pressure on top of the button, so I didn't want to change that and only added functionality for the FastEM without modifying the behaviour in other systems.

Comment (Sabrina85) [reply src/odemis/gui/cont/microscope.py:104]: Ok, I didn't know this. In your first version you had show_pressure = False, that is why I thought it should not be shown for the SecomStateController.

Comment (Sabrina85) [reply src/odemis/gui/cont/microscope.py:104]: I am now wondering how it works in the SPARC. There we also have a chamber...

**Comment by philipwinklerdelmic** [comment]
Resolved conflicts, @pieleric can you review this?

**Pull request merged**